frontend/src/pages/RecipientRecord/index.js

```diff
@@ -51,8 +52,9 @@ export function PageWithHeading({
               </div>
             ) : (
               <>
-                <h1 className={`ttahub-recipient-record--heading ${slug} page-heading ${headerMargin} margin-bottom-3`}>
+                <h1 className={`ttahub-recipient-record--heading ${slug} page-heading ${headerMargin} margin-bottom-3 display-flex`}>
                   {recipientNameWithRegion}
+                  {inlineHeadingChildren}
```

Comment:
It would be better for screenreader navigation to not have the button inside the h1